docs/self-managed/modeler/web-modeler/configuration.md

```diff
@@ -30,7 +30,52 @@ Web Modeler requires a PostgreSQL database as persistent data storage (other dat
 | `RESTAPI_DB_USER`     | Database user name     | `modeler-user`         |
 | `RESTAPI_DB_PASSWORD` | Database user password | \*\*\*                 |
 
-#### SMTP / email
+#### Configuring SSL for the database connection
+
+The generic way to configure an SSL connection between Web Modeler and the database is as follows:
+
+- Modify database connection URL `SPRING_DATASOURCE_URL` and customize connection parameters.
```

Comment:
Do these need to be performed in order? If so, I would adjust to a numerical list.

Reply:
Can be done in any order